autorest/adal/token.go

```diff
@@ -31,8 +32,11 @@ const (
 	// OAuthGrantTypeRefreshToken is the "grant_type" identifier used in refresh token flows
 	OAuthGrantTypeRefreshToken = "refresh_token"
 
-	// managedIdentitySettingsPath is the path to the MSI Extension settings file (to discover the endpoint)
-	managedIdentitySettingsPath = "/var/lib/waagent/ManagedIdentity-Settings"
+	// managedIdentitySettingsLinuxPath is the path to the MSI Extension settings file (to discover the endpoint)
+	managedIdentitySettingsLinuxPath = "/var/lib/waagent/ManagedIdentity-Settings"
+
+	// managedIdentitySettingsWindowsPath is the path to the MSI Extension settings file (to discover the endpoint)
+	managedIdentitySettingsWindowsPath = "C:/WindowsAzure/Config/ManagedIdentity-Settings"
```

Comment:
We shouldn't assume that the windows drive will always be "C" here.

Reply:
This info comes from the documentation

Reply:
Can it ever be something different, i.e. is this path user-configurable?  If so how does the end-user specify that?  I don't have a link to the docs.